gallery/src/styles/NxReadOnly/NxReadOnlyPage.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+const nxReadOnlyDlExampleCode = require('!!raw-loader!./NxReadOnlyDlExample.html').default;
+
+const NxReadOnlyPage = () => {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <p className="nx-p">
+          <code className="nx-code">.nx-read-only</code> is a design pattern for when you wish to display data in a read only form. It's basic structure is a label and one or more data points.
+        </p>
+        <p className="nx-p">
+          <code className="nx-code">.nx-read-only</code> is layed out using a description list
+          (<code className="nx-code">&lt;dl&gt;</code>) as the container.
+        </p>
+        <table className="nx-table nx-table--gallery-props">
+          <thead>
+            <tr className="nx-table-row">
+              <th className="nx-cell nx-cell--header">Class</th>
+              <th className="nx-cell nx-cell--header">Location</th>
+              <th className="nx-cell nx-cell--header">Details</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr className="nx-table-row">
+              <td className="nx-cell"><code className="nx-code">.nx-read-only</code></td>
+              <td className="nx-cell">Top level</td>
+              <td className="nx-cell">
+                This is the basic wrapper class. It is applied to the <code className="nx-code">&lt;dl&gt;</code>.
+              </td>
+            </tr>
+            <tr className="nx-table-row">
+              <td className="nx-cell"><code className="nx-code">.nx-read-only__label</code></td>
+              <td className="nx-cell">Element</td>
```

Comment:
Probably should say which element (the `dt`).  Same in the `.nx-read-only__data` section below.

Reply:
> Probably should say which element (the `dt`). Same in the `.nx-read-only__data` section below.

In both cases it says which element in the descriptive text that follows